### DISCUSSION TITLE
ElastAlert2 setup in Docker

### DISCUSSION THREAD

**Discussion by sajmo744** [jertel/elastalert2#1410] [Q&A] [answered]
Hi,

I am new to ElastAlert2 and never really setup Docker before.  Are there any good instructions out there that can help me setup ElastAlert2 in Docker container please on Linux OS ?

We are using Elasticsearch but without Watchers and would like to get the alerts ticketed into ServiceNow.  I see ElastAlert2 can do that.

Thanks

SM

**Comment by jertel** [comment, marked as answer]
Installing Docker on Linux is typically simple. But without knowing what type of Linux you're using I can't provide instructions. But really, this isn't the forum for requesting help on installing Docker. See #11.

Once you get Docker installed, follow the ElastAlert 2 instructions on running in Docker: https://elastalert2.readthedocs.io/en/latest/running_elastalert.html#as-a-docker-container
